README.md

```diff
@@ -1,10 +1,10 @@
-# Comms-deduplication
+# Mnemosyne
```

Comment:
why not `Moneta`? 😄

Reply:
The cat was walking on @filosganga's keyboard. Coin in Italian seems like a good name.